File: vy-android-app/app/src/main/java/com/lemon/vy3000/ui/ruter/RuterViewModel.java
package com.lemon.vy3000.ui.ruter;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class RuterViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public RuterViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is ruter fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: vy-android-app/app/src/main/java/com/lemon/vy3000/ui/fragment/ruter/RuterViewModel.kt
package com.lemon.vy3000.ui.fragment.ruter

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class RuterViewModel : ViewModel() {
    private val mText: MutableLiveData<String> = MutableLiveData()
    val text: LiveData<String>
        get() = mText

    init {
        mText.value = "This is ruter fragment"
    }
}